frontend/__mocks__/localStorage.ts

```diff
@@ -2,10 +2,10 @@ let _localStorage = {};
 
 (window as any).localStorage = (window as any).sessionStorage = {
   setItem(key, value) {
-    return Object.assign(_localStorage, {[key]: value});
+    Object.assign(_localStorage, { [key]: value} );
```

Comment:
I guess this _does_ adhere to the [real web API](https://developer.mozilla.org/en-US/docs/Web/API/Storage/setItem). But the existing way is what it _should_ be (in a FP world).

Reply:
Yup, the web storage API could be better :smiley: the above change just reflects the method signature in `node_modules/typescript/lib/lib.dom.d.ts`

```ts
setItem(key: string, value: string): void;
```